CMS/static/scss/crc.scss

```diff
@@ -5426,10 +5426,10 @@ h3 span a:hover, h3 span a:focus {
   color: #628e86
 }
 
-a:focus {
-  outline: 0;
-  text-decoration: none
-}
+// a:focus {
+//   outline: 0;
+//   text-decoration: none
+// }
```

Comment:
There's probably a lot of commented out code in this codebase but it is the code equivalent of littering. Fine for when you are developing. Clean it up before doing a PR

Reply:
(most of the litter was generated by us so apologies)